context.tf

```diff
@@ -20,6 +20,7 @@
 # will be null, and `module.this.delimiter` will be `-` (hyphen).
 #
 
+
```

Comment:
why this file has been changed?
It should be taken from https://github.com/cloudposse/terraform-null-label/blob/master/exports/context.tf